programs/autotension.cpp

```diff
@@ -256,208 +308,289 @@ void AutoTension<DOF>::init(ProductManager& pm, std::vector<int> args) {
 		hand = pm.getHand();
 		hand->initialize();
 		hand->close(Hand::GRASP);
-	}
-
-	// Joint 1 Tensioning will be added in and run simultaneously if present.
+	} 
+	================================================================== */
+
+	// ==================================================================
+	// JOINT POSITIONS TO MOVE TO DURING THE AUTOTENSIONING ROUTINE
+	// ==================================================================
+	// Joint1: This should be the same for both the arms.
+	// Initial Position: Fully stretched forward
 	jpInitial[0] = wam.getJointPositions();
-
-	// Joint 2
+	jpInitial[0][1] = 0.0;
+	jpInitial[0][2] = 0.0;
+	jpInitial[0][3] = 0.0;
+
+	// Start Position: Engage the Tang
+	jpStart[0] = jpInitial[0];
+	jpStart[0][0] = jpStopLow[0] + tangBuffer[0];
+	
+	// Move to one end
+	jpSlack1[0] = jpStart[0];
+	jpSlack1[0][0] = jpStart[0][0] - tangBuffer[0]/2.0;
+	
+	// Move to the other end
+	jpSlack2[0] = jpStart[0];
+	jpSlack2[0][0] = jpStopHigh[0] - tangBuffer[0] / 2.0;
+
+	// ==================================================================
+	// Joint 2: This is different for the two arms.
+	// Initial Position takes the arm to a safe location.
 	jpInitial[1] = wam.getJointPositions();
+	jpInitial[1][1] = 0.0; // Stretched forward
+	jpInitial[1][3] = jpStopHigh[3] - stopBuffer[3]; // Folded opposite side as HERB home
+
+	// Start Position: Engage the Tang
 	jpStart[1] = jpInitial[1];
-	jpStart[1][1] = jpStopHigh[1] - tangBuffer[1];
-	jpStart[1][2] = jpStopLow[2] + tangBuffer[2];
-	jpStart[1][3] = jpStopHigh[3] - stopBuffer[3];
+	if (armToAutotension == "left")
```

Comment:
This indentation is particularly misleading.

Reply:
Fixed, added clang-format with tabs.